schema/crdb/8.0.0/up2.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE omicron.public.bgp_config ADD COLUMN IF NOT EXISTS bgp_announce_set_id UUID NOT NULL;
+ALTER TABLE omicron.public.switch_port_settings_bgp_peer_config DROP COLUMN IF EXISTS bgp_announce_set_id;
```

Comment:
https://www.cockroachlabs.com/docs/stable/online-schema-changes#schema-changes-within-transactions

> Most schema changes should not be performed within an explicit transaction with multiple statements, as they do not have the same atomicity guarantees as other SQL statements. Execute schema changes either as single statements (as an implicit transaction), or in an explicit transaction consisting of the single schema change statement. 

Your changes here seem totally reasonable, but I think these separate "ALTER TABLE" statements count as distinct schema changes, and should be in separate files (so that they're in separate transactions).

https://www.cockroachlabs.com/docs/stable/online-schema-changes has a list of some exceptions, but I think this change applies to both up2.sql and up3.sql.

Reply:
This should  be straightened out in 9fcb8192c8b6106d0deff76bf179eac3c7f73db5